src/components/Header/TopNav/index.js

```diff
@@ -113,21 +118,30 @@ const TopNavBranding = ({ path, slug }) => (
         ? slug
         : getService({ slug, map: specialitiesToServicesMap })
 
+      const fillColorInitial = isSpecialityPage
+        ? getColor(specialities.nodes, slug) ||
```

Comment:
so getColor returns either a colour or... undefined?

Reply:
No but close: the colour is not a required field in Contentful and if it's not there, we still have the field but it returns null; in that case the logic that follows will still default back to `white`. Maybe we would want to make that field required in Contentful if a color is published.